docs/Installation.md

```diff
@@ -291,12 +291,30 @@ sudo install --mode=755 ./zm-backend.sh-postgresql /etc/init.d/zm-backend.sh
 mkdir "$HOME/logs"
 ```
 
-Install, configure and start database engine (and Perl bindings):
+The following block of commands is for **Debian 7** only. For all others, go to the step of installing
+database engine. First create or edit Debian 7 sources list file. Then fetch and import the repository signing key.
+And finally update the package lists.
+
+```sh
+sudo bash -c 'echo -e "\ndeb http://apt.postgresql.org/pub/repos/apt/ wheezy-pgdg main" >> /etc/apt/sources.list.d/pgdg.list'
+wget https://www.postgresql.org/media/keys/ACCC4CF8.asc -O /tmp/ACCC4CF8.asc

```

Comment:
A hardcoded tmp filename like this is a security concern, especially when running as root.

Reply:
@pawal I think it would be appropriate to state the reason for the security concern. Are your referring to the opportunity for an attacker to tamper with the file between the calls to `wget` and `apt-key`?

Reply:
No. Somebody can prepare a symlink for that tmp file and prepare an attack (now as root) for another file.
https://wiki.sei.cmu.edu/confluence/display/c/FIO21-C.+Do+not+create+temporary+files+in+shared+directories